pkg/installation/move.go

```diff
@@ -169,19 +169,32 @@ func moveToInstallDir(download, pluginDir, version string, fos []index.FileOpera
 	}
 
 	installPath := filepath.Join(pluginDir, version)
-	glog.V(2).Infof("Move %q to %q", tempdir, installPath)
-	if err = moveOrCopy(tempdir, installPath); err != nil {
+	glog.V(2).Infof("Move directory %q to %q", tempdir, installPath)
+	if err = moveOrCopyDir(tempdir, installPath); err != nil {
 		defer os.Remove(installPath)
 		return "", fmt.Errorf("could not rename file from %q to %q, err: %v", tempdir, installPath, err)
 	}
 
 	return installPath, nil
 }
 
-// moveOrCopy will try to rename a dir or file. If rename is not supported a manual copy will be performed.
-func moveOrCopy(from, to string) error {
+// moveOrCopyDir will try to rename a dir or file. If rename is not supported a
+// manual copy will be performed. Existing files at "to" will be deleted
+func moveOrCopyDir(from, to string) error {
 	// Try atomic rename (does not work cross partition).
-	err := os.Rename(from, to)
+	fi, err := os.Stat(to)
+	if err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("error checking move target dir %q: %+v", to, err)
```

Comment:
, err: %+v” also for other fmt calls

Reply:
I don't think the `err:` part is necessary.
stdlib or most 3rd party packages doesn't use this format. it's more like:

    something bad happened: [REASON]